com.github.bjaraujo.Bombermaaan.yml

```diff
@@ -0,0 +1,41 @@
+app-id: com.github.bjaraujo.Bombermaaan
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: bombermaaan
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --device=dri
+
+modules:
+  - shared-modules/glu/glu-9.json
+  - shared-modules/SDL/SDL-1.2.15.json
+  - shared-modules/SDL/SDL_mixer-1.2.12.json
```

Comment:
Please check if sdl12-compat works.

Reply:
I get this error https://github.com/bjaraujo/Bombermaaan/issues/46 at compile time even though I added `shared-modules/glu/glu-9.json` above it.